README.md

```diff
@@ -24,3 +24,22 @@ The SDK is tested against these versions of `esp-idf`:
 Previous versions of `esp-idf` are not guaranteed to work. If you find a problem using a later
 version of `esp-idf`, please [open an
 issue](https://github.com/astarte-platform/astarte-device-sdk-esp32/issues).
+
+## Notes on non volatile memory (NVM)
+
+The device Astarte credentials are by default stored in NVM. This means that credentials will be
```

Comment:
"The **device's** Astarte credentials are by default stored in **the** NVM." or "The device stores the Astarte credentials in the NVM by default"

Reply:
I agree with the suggestion and I applied the changes.
Just fyi, when speaking of memory I think it's ok to say a variable is stored in memory. Removing "the".